gsad/src/gsad_gmp.c

```diff
@@ -1019,65 +815,51 @@ response_from_entity (gvm_connection_t *connection, credentials_t *credentials,
   return res;
 }
 
-/**
- * @brief Get one resource, envelope the result.
- *
- * @param[in]  connection     Connection to manager
- * @param[in]  type           Type of resource.
- * @param[in]  credentials    Username and password for authentication.
- * @param[in]  params         Request parameters.
- * @param[in]  extra_xml      Extra XML to insert inside page element.
- * @param[in]  extra_attribs  Extra attributes for GMP GET command.
- * @param[out] response_data  Extra data return for the HTTP response.
- *
- * @return Enveloped XML object.
- */
 char *
-get_one (gvm_connection_t *connection, const char *type,
-         credentials_t *credentials, params_t *params, const char *extra_xml,
-         const char *extra_attribs, cmd_response_data_t *response_data)
+get_entity (gvm_connection_t *connection, const char *type,
```

Comment:
The get_entity function does not have any documentation comment.

Reply:
Fixed